autoPyTorch/utils/single_thread_client.py

```diff
@@ -61,8 +61,24 @@ def submit(
         func: Callable,
         *args: List,
         priority: int = 0,
+        key: Any = None,
+        workers: Any = None,
+        resources: Any = None,
+        retries: Any = None,
+        fifo_timeout: Any = "100 ms",
+        allow_other_workers: Any = False,
+        actor: Any = False,
+        actors: Any = False,
+        pure: Any = None,
```

Comment:
I think we also need to add `multi_objectives` to this list as the github actions raise  `E           TypeError: __init__() got an unexpected keyword argument 'multi_objectives'`

Reply:
It's unfortunately a seperate problem -_-

Reply:
It likely effects `autosklearn` as well so I'll update this PR when I find it

Reply:
okay thanks

Reply:
Think I fixed it, there are now timeouts occurring though which seem to be another seperate issue.